### DISCUSSION TITLE
Meross Power Strip MSP843P UK model compatible with Home Assistant?

### DISCUSSION THREAD

**Discussion by cwquek** [krahabb/meross_lan#352] [General]
Can anyone confirm whether the Meross Power Strip MSP843P UK model is supported with the Meross_lan integration in Home Assistant? Thanks.
https://www.meross.com/en-gc/smart-plug/smart-plug-socket/131

**Comment by krahabb** [comment]
Hello @cwquek,
Generally speaking evey new device has (almost) always worked 'out of the box' even if never tested before since the core protocol logic is being mantained by Meross among the various device generations.
Also, all of the smart switches flavors, even if radically different in hardware and form factors show exactly the same features-api from a software stand point (turn on/off that is)

In my opinion it should work 100% for all the classic basic functionalities (turn on/off, power readings-if device measures power-I cannot detect from commercial description) while some more specific behaviors could not be covered since they're just device specific and I've never managed to see one of these in action

You could be the first confirming (or not) it is working!

**Reply by cwquek** [reply]
Hello Krahabb, thank you for your prompt response. 👍
